packages/payment-integration-api/src/config/config.ts

```diff
@@ -91,6 +91,7 @@ export interface StoreCurrency {
 export interface CheckoutSettings {
     features: { [featureName: string]: boolean };
     checkoutBillingSameAsShippingEnabled: boolean;
+    checkoutUXSettings: { [settingName: string]: boolean };
```

Comment:
Does it make sense to have a list of `settingNames`? Since this can mean 
```
{
     checkoutUXSettings: {
          'test': true
     }
}
```
is valid?

Reply:
I think we treat it the same way as the feature flags. Your sample seems OK.
Otherwise, we need to update SDK every time we create a new option.

Reply:
Yes I think that should be ok, because we are adding other settings. Just need to be mindful. 
For feature flags its a list of strings so easier.